modules/ROOT/pages/import.adoc

```diff
@@ -12,7 +12,16 @@ You should use this tool when:
 
 * Import performance is important because you have a large amount of data (millions/billions of entities).
 * The database can be taken offline and you have direct access to one of the servers hosting your Neo4j DBMS.
-* The database is either empty or its content is unchanged since a previous incremental import.
+* The database is empty and you have to perform the initial data load.
+* You need to update your graph with large amount of data.
```

Comment:
The term "need" is used here instead of "have to"/"would like to". Should the same term representing the user intent be used in both sentences for consistency perhaps?